models/user.js

```diff
@@ -209,10 +209,6 @@ const UserSchema = new Schema(
         delete ret.__v;
         delete ret._id;
         delete ret.password;
-        delete ret.status.username.history;
-        delete ret.status.banned.history;
-        delete ret.status.suspension.history;
-        delete ret.tokens;
```

Comment:
Is keeping `tokens` a deliberate choice?

Reply:
It does not contain any critical data besides id's, and eventually might be a useful property.